doc/api/fs.md

```diff
@@ -2173,7 +2172,7 @@ Example:
 ```js
 fs.writeFile('message.txt', 'Hello Node.js', (err) => {
   if (err) throw err;
-  console.log('It\'s saved!');
+  console.log("It's saved!");
```

Comment:
Nit: the quote can be avoided if the message is something like `The file has been saved!`